base/compiler/ssair/passes.jl

```diff
@@ -818,12 +821,19 @@ end
     applyTvar = applyT.var
     applyTbody = applyT.body
```

Comment:
add comment here that `_lift_svec_ref` is only defined for `i == 1`?

Reply:
sure